lib/src/mapbox_maps_platform.dart

```diff
@@ -6,11 +6,18 @@ class _MapboxMapsPlatform {
   late final MethodChannel _channel = MethodChannel(
       'plugins.flutter.io', const StandardMethodCodec(), binaryMessenger);
```

Comment:
There should be suffix here, otherwise this method channel will broadcast all events to all active map instances.